atrium/models/account_number.py

```diff
@@ -0,0 +1,17 @@
+import json
```

Comment:
Do you need the json import here and in account owner? I would guess you only need this in atrium client since you're not using this here.

Reply:
Good catch, they have been removed 👍